server/tests/events.js

```diff
@@ -0,0 +1,399 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../app';
+import helper from './testHelper';
```

Comment:
if this is commented out or I go to the `user.js` test file to comment out the `export` and I run `npm test` most if not all the tests in this file will not pass.

So again, look into unit-testing